bandwidth-file-spec.txt

```diff
@@ -567,12 +567,17 @@ A.3. Generated by sbws version 1.X.X
 [TODO: change the version of sbws that generates this when it is implemented]
 1523911758
 version=1.2.0
-software=sbws
-software_version=1.X.X
 latest_bandwidth=2018-04-16T20:49:18
 file_created=2018-04-16T21:49:18
 generator_started=2018-04-16T15:13:25
 earliest_bandwidth=2018-04-16T15:13:26
+num_measured_relays=6000
```

Comment:
Please change the names of these keys to match the spec.

Otherwise, metrics will write parsing code using these names, and that code will break when sbws updates to match the spec.

Also change the names of the keys in "When there are not enough eligible measured relays:"